.gitignore

```diff
@@ -50,4 +47,8 @@ typings/
 .env
 
 # Editor configurations
-.vscode
+.vscode/
+
+# Target files
+lib/
+dist/
```

Comment:
should we add **/.DS_Store ? Those files always show up for me

Reply:
Really? I've never seen those. I guess the first person to run into can add it. I don't have any to verify that it's working